app/src/main/kotlin/co/touchlab/droidconandroid/MyActivity.kt

```diff
@@ -66,6 +66,12 @@ public class MyActivity : AppCompatActivity(), FilterInterface, NfcAdapter.Creat
             finish()
             return
         }
+        else if ( AppPrefs.getInstance(this).conventionStartDate != "" )
```

Comment:
Guessing this is just a temporary way to trigger the voting activity? I think kevin mentioned setting a hard date in BuildConfig before which the voting is shown, not sure if there are other ways to decide if we should go in to voting